addon/mixins/dirty-confirm-route.js

```diff
@@ -34,6 +34,9 @@ export default Ember.Mixin.create({
         case ROLLBACK: return true;
         case BUBBLE: return true;
       }
+    },
+    rollbackModel: function(model) {
+      this.checkModelForDirty(model, null, null, true);
```

Comment:
In the case of a modal, closing the modal is a specific action taken by the user.

Why would I always want to show the confirm?